specifyweb/frontend/js_src/lib/components/Molecules/Dialog.tsx

```diff
@@ -606,44 +616,63 @@ function useTitleChangeNotice(dimensionKey: string | undefined): void {
   }, [dimensionKey]);
 }
 
+const freezeSizeAfter = 2 * SECOND;
```

Comment:
When the code was copied, not all changes got copied - the original PR fixed the definition of `SECOND` which is not present in this PR